Formula/fop.rb

```diff
@@ -20,8 +14,13 @@ class Fop < Formula
   end
 
   def install
+    rm_rf Dir["fop/*.bat"] # Remove Windows files.
     libexec.install Dir["*"]
-    bin.write_exec_script libexec/"fop"
+
+    executable = libexec/"fop/fop"
+    executable.chmod 0555
```

Comment:
```
gstat -c "%a %n" /usr/local/opt/openssl/bin/openssl
555 /usr/local/opt/openssl/bin/openssl

gstat -c "%a %n" /usr/local/opt/botan/bin/botan
555 /usr/local/opt/botan/bin/botan
```
etc. I know `0755` tends to be used in formulae but I got the impression Homebrew corrected to `0555` behind the scenes.